01-syntax/04-high-order-functions/every-and-some/everyAndThenSome.js

```diff
@@ -1,7 +1,22 @@
 /* global every some */
-
-// → Your code here.
-
+function every(arr, method) {
```

Comment:
It's better to rename `method` to `func` or `fn`. Because by method we usually mean [object's methods](https://en.wikipedia.org/wiki/Method_(computer_programming))